frontend/javascripts/oxalis/api/api_latest.js

```diff
@@ -254,7 +254,7 @@ class TracingApi {
    * api.tracing.setActiveTree(3);
    */
   setActiveTree(treeId: number) {
-    const tracing = Store.getState().tracing;
+    const { tracing } = Store.getState();
```

Comment:
These changes were created when I saved this file. I think this is just an optimization made by the eslint/prettier. In all cases, it should not change the functionality of the methods.